recipes/genform/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-${CXX} ./src/*.cpp -o genform
+${CXX} ${CPPFLAGS} ${CXXFLAGS} ${LDFLAGS} ./src/*.cpp -o genform
```

Comment:
I forgot about/missed the missing flags before. We generally want to use them to make sure the compiled binaries are most compatible, e.g., run on older hardware.
Hope you don't mind me adding them.